Makefile

```diff
@@ -99,27 +99,40 @@ release: \
 
 release-victoria-metrics: \
 	release-victoria-metrics-amd64 \
```

Comment:
It looks like this target needs to be deleted after adding the `release-victoria-metrics-linux-amd64` target, since they perform the same action - build `victoria-metrics-prod` binary for (`GOOS=linux` + `GOARCH=amd64`).